SampleAPI.UnitTest/ControllerTests/EmployeeControllerTest.cs

```diff
@@ -0,0 +1,288 @@
+﻿using System.Data;
+using System.Drawing.Printing;
+using System.Linq;
+using AutoFixture;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using Moq;
+using SampleAPI.Constant;
+using SampleAPI.Controllers;
+using SampleAPI.Models;
+using SampleAPI.Repository;
+using Xunit;
+
+namespace SampleAPI.UnitTest.ControllerTests
+{
+    public class EmployeeControllerTest
+    {
+        public readonly Fixture _fixture;
+        private readonly Mock<ILogger<EmployeeController>> _loggerMock;
+        private readonly Mock<IEmployeeRepository> _employeeRepositoryMock;
+
+        public EmployeeControllerTest()
+        {
+            _fixture = new Fixture();
+            _loggerMock = new Mock<ILogger<EmployeeController>>();
+            _employeeRepositoryMock = new Mock<IEmployeeRepository>();
+        }
+
+        private EmployeeController CreateController()
```

Comment:
do you really need a method to get an instance of the controller, or could this simply be a priavate field on the test class and initialized in the constructor ?

Reply:
Yes, we can have it as private field.